services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -244,3 +270,43 @@ export function getValidationFromFormTemplate(reportJson: ReportJson) {
 export function getPossibleFieldsFromFormTemplate(reportJson: ReportJson) {
   return Object.keys(getValidationFromFormTemplate(reportJson));
 }
+
+const routesToIncludeInPCCM = {
+  "A: Program Information": [
+    "Point of Contact",
+    "Reporting Period",
+    "Add Plans",
+  ],
+  "B: State-Level Indicators": ["I: Program Characteristics"],
+  "C: Program-Level Indicators": ["I: Program Characteristics"],
+  "D: Plan-Level Indicators": ["I: Program Characteristics", "VIII: Sanctions"],
+  "Review & Submit": [],
+} as { [key: string]: string[] };
+
+const entitiesToIncludeInPCCM = ["plans", "sanctions"];
```

Comment:
routes to include from ticket

> 
    Section A: Program Information
        Point of Contact
        Reporting Period
        Add Plans
    Section B: State-Level Indicators
        Program Characteristics and Enrollment
    Section C: Program-Level Indicators
        Program Characteristics
    Section D: Plan-Level Indicators
        Program Characteristics and Enrollment
        Sanctions
    Review and Submit

entities inferred from same list

Reply:
Comment: I think that, strictly speaking, the entity inference could have been done in code rather than hardcoding `plans` and `sanctions` in this array. But that code would have been terribly complicated compared to this one-line declaration, so I completely agree with your approach.